internal/datastore/crdb/crdb_test.go

```diff
@@ -135,6 +139,7 @@ func TestWatchFeatureDetection(t *testing.T) {
 	}
 	for _, tt := range cases {
 		t.Run(tt.name, func(t *testing.T) {
+			t.Parallel()
```

Comment:
any time you do this I think you want to be careful to make a copy of the iteration variable (i.e. `tt := tt`) Now that these can run in parallel, `tt` will move forward before all of one iteration's calls have run (likely will just reference the last case)

Reply:
oh, that's a good catch!

Reply:
The `paralleltest` linter I just introduced should catch these. Unfortunately the datastore tests do not start with `Test*` and so the linter does not process them. I fixed the instances it missed in https://github.com/authzed/spicedb/pull/910/commits/4d9dfce52a846487a798d1e9317d5a2a700ee8b9